### DISCUSSION TITLE
How to remove urls from the info box from the display results?

### DISCUSSION THREAD

**Discussion by linlinxza** [searxng/searxng#4152] [Q&A] [answered]
I would like to remove the social media junk from the info box of the search results:

![image](https://github.com/user-attachments/assets/51bbbcab-b94c-41e9-baa6-63bad405a47d)

How do I go about doing this in the settings.yml file?

**Comment by return42** [comment, marked as answer]
> How do I go about doing this in the settings.yml file?

This is not possible.

**Reply by linlinxza** [reply]
Seriously? Maybe this should be a feature.

**Reply by return42** [reply]
Related: https://github.com/searxng/searxng/issues/1348
